### PR TITLE
Map loader optimizations

### PR DESCRIPTION
**Pull request by Cyberboss**
This time it works

### CODE REVIEW

File: code/modules/mapping/reader.dm [hunk 1]
```diff
@@ -225,6 +244,18 @@ var/global/dmm_suite/preloader/_preloader = new
...
 		while(dpos != 0)
 
+		//check and see if we can just skip this turf
+		//So you don't have to understand this horrid statement, we can do this if
+		// 1. no_changeturf is set
+		// 2. the space_key isn't set yet
+		// 3. there are exactly 2 members
+		// 4. with no attributes
+		// 5. and the members are world.turf and world.area
+		if(no_changeturf && !space_key && members.len == 2 && members_attributes.len == 2 && length(members_attributes[1]) == 0 && length(members_attributes[2]) == 0 && (world.area in members) && (world.turf in members))
```
Comment (RemieRichards) [inline code/modules/mapping/reader.dm]: so, ```if it matches "???" = (/turf/base, /area)```

Comment (Cyberboss) [inline code/modules/mapping/reader.dm]: basically

Comment (RemieRichards) [inline code/modules/mapping/reader.dm]: include (something similar) it in the comment, all people reading the maploader should understand map models.

**Comment by optimumtact** [comment]
Saving this for later

**Pull request merged**